.swiftlint.yml

```diff
@@ -0,0 +1,155 @@
+opt_in_rules:
+- attributes
+- closure_end_indentation
+- closure_spacing
+- empty_count
+- explicit_init
+- fatal_error_message
+- file_header
+- first_where
+- implicit_return
+- implicitly_unwrapped_optional
+- nimble_operator
+- no_extension_access_modifier
+- number_separator
+- operator_usage_whitespace
+- overridden_super_call
+- private_outlet
```

Comment:
Since this project doesn't even link against a UI framework, this can be removed.